VMs/metrics/6_start_clone_agent.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+JARSRC="${HOME}/bin/agent.jar"
+JARDEST_DIR="/var/lib/jenkins"
+JARDEST="${JARDEST_DIR}/agent.jar"
+
+LOGFILE=/dev/null
+
+source global_env.sh
+
+die() {
+	echo "Fatal: $*" >&2
+	exit 1
+}
+
+error() {
+	echo "Error: $*" >&2
+}
+
+message() {
+	echo "Message: $*"
+}
+
+main()
+{
+	if [ ! -f ${CONFIG_DIR}/${ENV_FILE} ]; then
+		die "Env file ${CONFIG_DIR}/${ENV_FILE} missing"
+	fi
+
+	source ${CONFIG_DIR}/${ENV_FILE}
+
+	# Ensure we do not have an existing instance
+	message "Ensuring ${CONFIG_DIR} is shut down"
+	./7_delete_clone.sh ${CONFIG_DIR}
+
+	# And make the clone
+	message "Cloning ${VM_NAME} -> ${VM_CLONE_NAME}"
+	./4_clone_vm.sh ${CONFIG_DIR}
```

Comment:
I was assuming these scripts had a numeric prefix so they were run in that order. But if so, why is `7` run before `4`?